packages/react-components/react-tabster/src/focus/createCustomFocusIndicatorStyle.ts

```diff
@@ -28,6 +28,8 @@ export const createCustomFocusIndicatorStyle = (
     [`:global(.fui-FluentProvider)`]: {
       [`& .${FOCUS_VISIBLE_CLASS}`]: style,
     },
```

Comment:
Can this be wrapped in a `@supports not selector(:focus-visible)` block so that it's easier for the browser to know they don't have to try and match this style if `:focus-visible` is supported?